packages/expo-updates/scripts/createManifest.js

```diff
@@ -20,6 +20,9 @@ const filterPlatformAssetScales = require('./filterPlatformAssetScales');
     projectRoot = path.resolve(possibleProjectRoot, '..');
   }
 
+  // remove any relative '..' segments from project root
+  projectRoot = path.resolve(projectRoot);
```

Comment:
should we move this up to line 18? so:

```
projectRoot = path.resolve(possibleProjectRoot);
```

it seems like this is doing the same thing but a little bit later on and possibly on an undefined value